### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by Itswali**
Project requirements

    Create a method called current_user in your ApplicationController to make current user data available to all controllers.
        It will return the first user from the database.
        Don't worry about proper authentication at this time. You will add it in the upcoming projects.
    Create forms to perform the following functions:
        Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
        Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
    Allow Users to add likes to Posts.

### CODE REVIEW

File: db/migrate/20230616212030_add_user_to_likes.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, foreign_key: true
+  end
+end
```
Comment (BenMukebo) [inline db/migrate/20230616212030_add_user_to_likes.rb]: - [ ] Kindly remove this migration as it's throwing errors. you are trying to add the user reference in like but this was already added.
image

File: db/migrate/20230616212113_remove_duplicate_user_id_from_likes.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class RemoveDuplicateUserIdFromLikes < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :likes, :user_id
+  end
+end
```
Comment (BenMukebo) [inline db/migrate/20230616212113_remove_duplicate_user_id_from_likes.rb]: - [ ] Kindly remove this migration as it's throwing errors. you are trying to remove the user reference in like but this does not exist in likes.
image

File: db/migrate/20230617192156_add_user_id_to_likes.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserIdToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, foreign_key: true
+  end
+end
```
Comment (BenMukebo) [inline db/migrate/20230617192156_add_user_id_to_likes.rb]: - [ ] Kindly remove this migration as it's throwing errors. you are trying to add the user reference in like but this was already added.
image

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Itswali,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- You've fixed all linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Professional README ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @Itswali  👋 

## Things to Appreciate
- Good job for deleting files requested by the previous Code Reviewer that were causing errors. 👍 

## [OPTIONAL]
- Please add a descriptive Heading for PR Summary 👍 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**